pytorch_blade/torch_blade/clustering/support_fusion_algorithm.py

```diff
@@ -189,7 +190,9 @@ def can_merge(u, v):
         return True
 
     graph_topolist = graph_builder.group_topolist()
-    max_iter_count = 10
+    # some graph unions may not converge in 10 iterations, provide customize setting from env
+    # TODO: refine cluster policy
+    max_iter_count = int(os.getenv('TORCH_CLUSTER_MAX_ITER_COUNT', 10))
```

Comment:
I think a config option is torch_blade's convention.

Reply:
Good point, done.